source/manual/github.html.md

```diff
@@ -34,46 +34,45 @@ There are several GOV.UK GitHub teams within alphagov, including:
   This team allows the user to create a branch in a GOV.UK repository and open a pull request, but only a developer can merge the request.
   Only content designers should be added to this team.
 
-# Getting access to GitHub
+## Getting access to GitHub
 
-Not everyone on GOV.UK requires GitHub access, as much of what we do is in the open. However, if your role requires it, you should be added to the org and the relevant team(s) through Terraform, in [govuk-user-reviewer][govuk-user-reviewer] - _not_ manually added through the GitHub UI itself, as this breaks the Terraform setup. Note that you will be sent an invitation email and will have to accept the invite before you are added to the organisation.
+If your role requires access to Github, you'll need to be added to the organisation and the relevant team(s) through Terraform, in [govuk-user-reviewer][govuk-user-reviewer] - _not_ manually added through the GitHub UI itself, as this breaks the Terraform setup. Note that you will have to accept the invite sent via email before you're added to the organisation.
 
-- If you're a content designer, ask for GitHub access via Zendesk (see [example ticket](https://govuk.zendesk.com/agent/tickets/5812930/events)). Ensure to include `govuk_platform_support` tag.
-- If you're an engineer or contractor, ask your tech lead to follow the instructions in [govuk-user-reviewer][govuk-user-reviewer] to add you.
-- If you don't have a tech lead, [ask someone in Senior Tech](/manual/ask-for-help.html#contact-senior-tech) to add you. You must state
-  - your role
-  - which team you're in
-  - your GitHub handle
-  - which GitHub team(s) you should join ([see list](#govuk-teams))
-  - why you need access
+- If you're an engineer or contractor, ask your tech lead to add you by following the instructions in [govuk-user-reviewer][govuk-user-reviewer].
+- If you don't have a tech lead, [ask someone in Senior Tech](/manual/ask-for-help.html#contact-senior-tech) to add you.
+- If you're a content designer, ask for GitHub access via Zendesk (see [example ticket](https://govuk.zendesk.com/agent/tickets/5812930)). Make sure you include the `govuk_platform_support` tag.
 
-## Removing access to GitHub
-
-Users are removed from the GitHub organisation when their entry in [govuk-user-reviewer][govuk-user-reviewer] is deleted.
+In these last two cases, you must state:
 
-# GOV.UK repos
+- your role
+- which team you're in
+- why you need access
+- your GitHub username
+- which GitHub team(s) you should join ([see list](#govuk-teams))
```

Comment:
```suggestion
- which GitHub team(s) you should join ([see list](#gov-uk-teams-in-github))
```
The `id` will change after the header is updated from `GOV.UK teams` to `GOV.UK teams in Github`